components/card/card.css

```diff
@@ -0,0 +1,106 @@
+@import url("/docs/main.css");
+
+.card {
+    min-height: fit-content;
+    width: 60%;
+    justify-content: flex-start;
+    box-shadow: 2px 2px 10px grey;
+    border-radius: 5px;
+    border: 3px solid var(--shadow-color);
+    margin: 1rem;
+}
+
+.card_container {
+    display: flex;
+    flex-direction: column;
+    margin: 1rem 1rem 0 1rem;
+}
+
+.card_image {
+    display: flex;
+}
+
+.card_vertical {
+    display: flex;
+    flex-direction: column;
```

Comment:
can add seperate css class `flex-col` as `display:flex` `flex-direction:column` is repeating

Reply:
Done, Vaibhav!